waku/v2/node/config.nim

```diff
@@ -76,10 +77,14 @@ type
       name: "rln-relay" }: bool
     
     rlnRelayMemIndex* {.
-      desc: "(experimental) the index of node in the rln-relay group: a value between 0-49 inclusive",
-      defaultValue: 0
-      name: "rln-relay-membership-index" }: uint32
-    
+      desc: "(experimental) the index of node in the rln-relay group: a value between 0-99 inclusive",
+      defaultValue: MembeshipIndex(0)
+      name: "rln-relay-membership-index" }: MembeshipIndex
```

Comment:
I missed this during the first reading, but perhaps now is the time to fix the `MembeshipIndex` type to `MembershipIndex` (missing `r`)? :)

Reply:
:D done in https://github.com/status-im/nim-waku/pull/722/commits/58589923801d2dbef6463367f76275718f8f5f64